.github/workflows/dp-workflow.yml

```diff
@@ -390,6 +409,8 @@ jobs:
           path: |
             ${{ env.TEST_DIR }}
             /tmp/${{ matrix.test_type }}.txt
+            /tmp/k8s-pods-logs
```

Comment:
NIT: is this more robust if results files are collected in a dedicated /tmp/foo folder and this folder is uploaded? I.e., omit to list all files created in different places.

Reply:
Good point. I will improve it in another PR.